### DISCUSSION TITLE
Enable command line only usage on Linux

### DISCUSSION THREAD

**Discussion by Civitello** [Zodsmar/SeedSearcherStandaloneTool#128] [Ideas]
For maximum performance it would be nice to be able to run this fun the command line without needing anything that supports a GUI.

**Comment by Zodsmar** [comment]
How would you like that work? Like how would you set all the presets to run from the command line? If I make a config file that can be exported (essentially all of the settings) that can be loaded in as an argument from the command line would that work? You would need to first configure it in the UI and export the config file, the run it from the command line with that file as an argument.

Would that work for you? Or did you have another idea in mind

**Reply by Civitello** [reply]
A config file would be perfect!
